go.mod

```diff
@@ -1,16 +1,18 @@
-module github.com/uber/athenadriver
+module github.com/launchdarkly-labs/athenadriver
```

Comment:
Leaving notes where we have references to launchdarkly/LD. Will need to remember to change these when attempting to merge back to the uber repo.